### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by parham-dev27**
Made it look better. If you have any suggestions let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,137 @@
...
+1. Build the Project
+    ```sh
+    make
+    ```
+
+## Usage
+
+-   Help Page:
+    -   `./hexit -h`
+-   Display the hex dump of a file.:
```
Comment (marprok) [inline README.md:110]: The `.` at the end of the sentence is not needed here.

File: README.md [hunk 2]
```diff
@@ -1,36 +1,137 @@
...
+
+## About The Project
+
+[product-demo]: https://github.com/marprok/hexit/assets/18293204/73c7c699-9d60-40c1-8f6a-911c67d5514f
+
+![Product Name Screen Shot][product-demo]
+
+-   <strong>Hexit</strong> is a easy to use program used to view and edit binary files.
+-   Hexit allows editing the raw data contents of a file, instead of other programs which attempt to interpret the data for you.
+-   Hexit has three areas: an address area on the left, a hexadecimal area in the center, and a character area on the right.
```
Comment (marprok) [inline README.md:65]: The word 'area' is a bit generic. I would propose to rephrase the start of this sentence as such:
`Hexit's UI is separated into three visual areas: ...`

**Review by marprok — CHANGES_REQUESTED** [review]
Just some minor comments. Once you resolve them we can merge this into master :)

**Comment by parham-dev27** [comment]
Resolved. Check new commit

**Comment by marprok** [comment]
Thank you very much for spending some of your time to improve README, it looks great!

**Review by marprok — APPROVED** [review]

**Pull request merged**